resources/img/wizard/keyword-subcategory-dam_break.svg

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Compress it to make the size smaller

Reply:
This SVG is not in the correct folder. It must be in **safe**/resources/img/wizard/keyword-subcategory-dam_break.svg
Then Travis will be happy

Reply:
i see.. here i push my update..